### PR TITLE
[formidable] Fix incorrect argument to fileWriteStreamHandler and add missing newFilename property in FileJSON

### PR DESCRIPTION
**Pull request by gboer**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/node-formidable/formidable/issues/821

### CODE REVIEW

File: types/formidable/VolatileFile.d.ts [hunk 1]
```diff
@@ -1,5 +1,14 @@
-import PersistentFile = require("./PersistentFile");
+import { EventEmitter } from 'events';
```
Comment (peterblazejewicz) [inline types/formidable/VolatileFile.d.ts:1]:
```suggestion
import { EventEmitter } from 'node:events';
```
(keep with upstream)

File: types/formidable/VolatileFile.d.ts [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+import { File, FileJSON } from "./";
```
Comment (peterblazejewicz) [inline types/formidable/VolatileFile.d.ts:2]:
```suggestion
import type { File, FileJSON } from "./index";
```

Comment (gboer) [reply types/formidable/VolatileFile.d.ts]: Fixed :)

File: types/formidable/VolatileFile.d.ts (hunk 1 above)
Comment (gboer) [reply types/formidable/VolatileFile.d.ts]: Fixed :)

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@gboer thanks!

**Comment by gboer** [comment]
Ready to merge

**Pull request merged**